### ISSUE TITLE
Tx Gas Limit

### ISSUE THREAD

**Issue by chevdor** [MetaMask/metamask-plugin#726]
I can send ETH from my account to another *regular account*.
However, if I try sending ETH from my account to a contract account, the tx exceeeds the gas limit and fail.

Is there a way to set the tx gas limit for Metamask?

Here are some sample data (on testrpc).
Sending to a contract:
```
 Transaction: 0xdf6401c56a89d0dfa52c9f6fac8b434c3d11b1e3482a9f22b550a7dff9e069c2
  Gas usage: 0x72e1
  Block Number: 0x12
  Block Time: Wed Oct 12 2016 11:31:10 GMT+0200 (CEST)
  Runtime Error: out of gas
```
Sending to an account:
```
 Transaction: 0xbb7b68f832cb463b8b41fbb40d816a4aa99939301dbbebf0e35bb3fcf2e64fb6
  Gas usage: 0x5208
  Block Number: 0x13
  Block Time: Wed Oct 12 2016 11:35:43 GMT+0200 (CEST)
```

NOTE: I am using testrpc 3.0. In this version, there is a major change regarding the gas limit per transaction that is now 90k by default (I think it used to be 4.7M).

I see an issue that may be related: #60

**Comment by chevdor** [comment]
#720 probably brings a solution to that problem

**Comment by chevdor** [comment]
#720 does not help as it proposes a higher gas price but not a higher amount of gas.

**Comment by kumavis** [comment]
@chevdor can you try against morden instead of testrpc and see if you still have the same problem?
`eth_estimateGas` is not performed by MetaMask, its handled by the rpc client. More likely this is an issue in testrpc/provider-engine.

**Comment by kumavis** [comment]
kumavis @kumavis 12:52
> how does mist autofill tx gasLimit ?
Fabian Vogelsteller @frozeman 12:55
> Running estimate gas and then adding 100000

**Comment by flyswatter** [comment]
Either solved by #720, or will be solved by #738, so closing in favor of those.
